tests/test_validators.py

```diff
@@ -154,48 +146,38 @@ def test_validate_whole():
     class Model(BaseModel):
         a: List[int]
 
-        @validator('a', mode='before')
-        def check_a1(cls, v: Any):
+        @field_validator('a', mode='before')
+        @classmethod
+        def check_a1(cls, v: List[Any]) -> List[Any]:
             v.append('123')
             return v
 
-        @validator('a')
-        def check_a2(cls, v: Any):
+        @field_validator('a')
+        @classmethod
+        def check_a2(cls, v: List[int]) -> List[Any]:
             v.append(456)
             return v
 
     assert Model(a=[1, 2]).a == [1, 2, 123, 456]
 
 
-@pytest.mark.xfail(reason='working on V2')
-def test_validate_kwargs():
```

Comment:
did you retain any each_item tests? If not, I'm thinking we should keep this with a `with pytest.warns(DeprecationWarning)` wrapper or similar

Reply:
I did retain the `each_item` tests. This test is weird because it's supposed to be testing `**kwargs` (which is no longer supported and in fact we have / will have in a minute tests proving errors) but is also using `each_item` for no apparent reason. So I'm removing this test in favor of tests that test each one of those things on their own. Unless you see a relationship between `**kwargs` and `each_item` that I am missing.